tests/functional/test_change_email_address.py

```diff
@@ -0,0 +1,38 @@
+from flask import url_for
+
+supplied_email = 'some.one@digital.cabinet-office.gov.uk'
+
+
+class When_on_change_email_address_page(object):
+
+    def it_shows_the_email_address_supplied_in_the_textbox(
+            self, live_server, browser):
+
+        browser.visit(url_for(
+            'main.change_email_address',
+            email_address=supplied_email,
+            _external=True))
```

Comment:
Actually, this is being refactored, probably best to pull this PR and put changes in the refactored change email PR